client/app/admin/create-project/create-project.controller.js

```diff
@@ -433,11 +449,25 @@
                     vm.createProjectSuccess = true;
                     // Navigate to the edit project page
                     $location.path('/admin/edit-project/' + data.projectId);
-                }, function () {
+                }, function (reason) {
                     vm.waiting = false;
                     // Project not created successfully
                     vm.createProjectFail = true;
                     vm.createProjectSuccess = false;
+
+                    switch (reason.status) {
+                        case 400:
+                            vm.createProjectFailReason = reason.data.error
+                            break;
+                        case 500:
+                            vm.createProjectFailReason = reason.data.error
+                            break;
+                        default:
+                            vm.createProjectFailReason = 'The project could be be created. This may be because the combined data size of the AOI and the ' +
+                            'grid is too large. Try again with a smaller AOI, fewer tasks and/or larger tasks.'
```

Comment:
Again, we can't have any text in the controller for translation reasons. We also can't use any error messages coming from the server.